GestorDeEstoque/Models/User.cs

```diff
@@ -0,0 +1,16 @@
+namespace GestorDeEstoque.Models
+{
+  public class Users
```

Comment:
A classe deveria chamar User igual o nome do arquivo.
Só questão de padrão mesmo.

```suggestion
  public class User
```